### ISSUE TITLE
nnx.pmap documentation seems to be for vmap

### ISSUE THREAD

**Issue by hrbigelow** [google/flax#4330]
Hi, 

The [source code documentation](https://github.com/google/flax/blob/c12ba53eb5d4e09913ffe575768d6ef20ce2698d/flax/nnx/transforms/iteration.py#L441-L520) for `nnx.pmap` appears to be instead referring to nnx.vmap.

**Comment by cgarciae** [comment]
This is correct. Sorry it was adapted very quickly from `vmap` for an experiment.
Will update soon.

**Comment by hrbigelow** [comment]
No problem, thank you @cgarciae

**Comment by HyperBlaze456** [comment]
will there be any updates soon...?
